src/coreclr/clrdefinitions.cmake

```diff
@@ -169,9 +169,9 @@ else()
   add_compile_definitions($<$<BOOL:$<TARGET_PROPERTY:CROSSGEN_COMPONENT>>:FEATURE_PREJIT>)
 endif(FEATURE_PREJIT)
 
-if(CLR_CMAKE_TARGET_WIN32 OR CLR_CMAKE_TARGET_LINUX)
+if(CLR_CMAKE_TARGET_WIN32 OR CLR_CMAKE_TARGET_LINUX OR CLR_CMAKE_TARGET_DARWIN)
```

Comment:
Should this `if` be deleted to enable this unconditionally?

We can still keep `FEATURE_PROFAPI_ATTACH_DETACH` around in case we want to disable it in some cases in future.

Reply:
@jkotas I thought about this, but decided not to do it without checking on FreeBSD and others (if any). Shall I?

Reply:
I do not think you need to validate it. Given that it worked on OSX without any extra changes, it is very likely that it works everywhere just fine.